res/css/views/rooms/_EventTile.scss

```diff
@@ -124,7 +132,7 @@ $threadInfoLineHeight: calc(2 * $font-12px); // See: _commons.scss
             padding-top: 1px;
             padding-bottom: 3px;
             padding-left: $left-gutter;
-            line-height: $font-22px;
+            line-height: var(--EventTile_group_line-line-height);
```

Comment:
Instead of manually setting the line heights for each individual layout, would it be possible to just remove this line height declaration, and let the line height cascade from the `mx_EventTile_line` element?

Reply:
Yes, I still recommend that we remove the declaration. Adding more of them might reduce the chances of this change directly causing regressions, but in the long term it would be more fragile (https://github.com/vector-im/element-web/issues/22687 being an example of exactly that). So in the interest of simplifying the timeline CSS I say we go with the option of removing it.

Reply:
That is fair enough. I just would like to ask everyone to avoid putting the responsibility on me in case regressions would happen :-D